departments.html

```diff
@@ -0,0 +1,215 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    
+    <title>Paediatric Website</title>
+
+    <!-- Bootstrap CSS -->
+     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+     <!--fontawesome 5-->
+ <script src="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/js/all.min.js"></script>
+     <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    
+ <!-- Navigation -->
+ <nav class="navbar navbar-expand-lg  fixed-top">
+  <div class="container">
+    <a class="navbar-brand" href="#">
+      <h5> <span class="brand-logo">New Life</span> <i class="fas fa-smile logo"></i></h5> 
+        </a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+    <div class="collapse navbar-collapse" id="navbarResponsive">
+      <ul class="navbar-nav ml-auto">
+        <li class="nav-item ">
+          <a class="nav-link" href="home.html">Home
+                
+              </a>
+        </li>
+        <li class="nav-item ">
+          <a class="nav-link" href="about.html">About</a>
+          
+        </li>
+        <li class="nav-item active">
+          <a class="nav-link" href="departments.html">Departments</a>
+          <span class="sr-only">(current)</span>
+        </li>
+       
+        <li class="nav-item">
+          <a class="nav-link" href="contact.html">Contact</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+</nav>
+
+<section id="dept">
+  <div class="container">
+    <div class="row ">
+      <div class="col-md-12 ">
+        
+          
+       
+         
+      </div>
+    </div>
+  </div>
+  
+  </section>
+<section id="dept-main">
```

Comment:
The cards do not seem to be responsive when the browser window is resized.

Reply:
Departments page responsive. 
Please check.